.github/workflows/test.yaml

```diff
@@ -0,0 +1,47 @@
+name: Test
+
+on:
+  workflow_call:
+    null
+
+jobs:
+  integration-test:
+    runs-on: ubuntu-latest
```

Comment:
Use `jobs.*.container` instead of loading artifact and running container manually. We can push the build image to the GitHub container registry. See dataaxiom/ghcr-cleanup-action@v1 for a nice action to clean up the registry.